sources/packages/api/src-sql/StudentPDVerified/Add-StudentPDVerified.sql

```diff
@@ -0,0 +1,13 @@
+
+-- Add the column with pd_verified.
+ALTER TABLE students
```

Comment:
I believe that there is a problem with the syntax.
The correct command would be:
ALTER TABLE students
    ADD COLUMN IF NOT EXISTS pd_verified BOOLEAN;